### ISSUE TITLE
Potentially erroneous message when resuming from suspended  

### ISSUE THREAD

**Issue by MarkyMarkMcDonald** [vmware-archive/pcfdev#109]
We noticed PCF dev was down after running for a few days straight. We checked the status and noticed it was suspended. We tried resuming once, and saw a warning about not enough free memory detected.  We killed a few memory hogs and tried resuming again, but encountered the message again - with VBoxHeadless running at 4gigs still. 
We decided to destroy and start the image instead of resuming with the warning (Successfully).

We're not sure how high priority / if this is a problem, please feel free to close this issue if not applicable.

```
○ → cf pcfdev status
Suspended

○ → cf pcfdev resume

Less than 4096 MB of free memory detected, continue (y/N): > N


○ → cf pcfdev resume

Less than 4096 MB of free memory detected, continue (y/N): > N


○ → cf pcfdev destroy
PCF Dev VM has been destroyed


○ → cf pcfdev start
Using existing image
Allocating 4096 MB out of 8192 MB total system memory (5237 MB free).
```

**Comment by sclevine** [comment]
Hi @MarkyMarkMcDonald,

Thanks for the report! We will look into this and get back to you.

**Comment by ekcasey** [comment]
@MarkyMarkMcDonald 

Right now pcfdev treats both paused and saved virtualbox vms as "suspended". However, virtualbox releases the memory of a saved vm but not a paused vm. Our current memory check assumes that a suspended vm is saved and checks to make sure it has enough memory to start. It think your vm is paused instead, and therefore already has the reserved memory it requires. We added a story to the backlog to skip the memory check for paused vms.

**Comment by sclevine** [comment]
Fixed in 0.20.0.
